requirements/quality.txt

```diff
@@ -336,21 +342,21 @@ pylint-plugin-utils==0.6
     # via
     #   pylint-celery
     #   pylint-django
-pylint==2.8.3
+pylint==2.9.6
```

Comment:
I anticipate adding a constraint for the pylint version would suffice to address this. Feel free to create an AU ticket to clean up if you choose to go that route.